Description: Fix hello.1 (manual generated by help2man) problem on Launchpad

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2024-04-18

--- hello-2.10.orig/Makefile.am
+++ hello-2.10/Makefile.am
@@ -45,12 +45,12 @@ DEFS = -DLOCALEDIR=\"$(localedir)\" @DEF
 
 CLEANFILES =
 
-if !GIT_CROSS_COMPILING
+#if !GIT_CROSS_COMPILING
 man_MANS = hello.1
 EXTRA_DIST += $(man_MANS)
-endif
+#endif
 
-# if BUILD_FROM_GIT
+#if BUILD_FROM_GIT
 hello.1: hello
 	$(HELP2MAN) --include=$(top_srcdir)/man/hello.x $(top_builddir)/hello -o $@-t
 	chmod a=r $@-t
